### PR TITLE
[Ubuntu] Migrate tests for common tools to Pester (part 1)

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
In scope of this PR we are migrating tests for common tools to Pester (see list of tools in related issue).

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by vsafonkin — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/installers/clang.sh [hunk 1]
```diff
@@ -41,14 +33,16 @@ function SetDefaultClang {
...
+    versions=$(jq -r '.Clang.versions[]' $toolset)
+    default_clang_version=$(jq -r '.Clang.default_version' $toolset)
 fi
 
 if isUbuntu20 ; then
-    versions=( "8" "9" "10" )
-    default_clang_version="10"
+    versions=$(jq -r '.Clang.versions[]' $toolset)
+    default_clang_version=$(jq -r '.Clang.default_version' $toolset)
 fi
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/clang.sh:46]: Why do we need these conditions?
Code is the same for all platforms

File: images/linux/toolsets/toolset-1604.json [hunk 2]
```diff
@@ -213,5 +213,26 @@
             "3.1",
             "5.0"
         ]
+    },
+    "Clang": {
```
Comment (maxim-lobanov) [inline images/linux/toolsets/toolset-1604.json:217]: Please use lower camel case for naming `clang`, `gcc`

Comment (dibir-magomedsaygitov) [reply images/linux/toolsets/toolset-1604.json]: Thank you, changed to lower case

File: images/linux/scripts/installers/clang.sh (hunk 1 above)
Comment (dibir-magomedsaygitov) [reply images/linux/scripts/installers/clang.sh]: Thank you, fixed now

File: images/linux/scripts/installers/ansible.sh [hunk 3]
```diff
@@ -17,8 +18,4 @@ fi
 apt-get install -y --no-install-recommends ansible
 
 # Run tests to determine that the software installed as expected
```
Comment (al-cheb) [inline images/linux/scripts/installers/ansible.sh:20]: Should we remove the comment?

Comment (dibir-magomedsaygitov) [reply images/linux/scripts/installers/ansible.sh:20]: Thank you, I'll remove it as well

**Review by maxim-lobanov — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**